examples/plot_pp_study.py

```diff
@@ -0,0 +1,362 @@
+# example, assume that if the model’s output is above the defined
+# threshold, the described area is more heavily policed. Some key
+# characteristics of the scenario:
+#
+#
+# * Stakeholders: police officers, members of the community (especially
+#   people who may be more affected by predictive policing than others),
+#   contractor / data scientist
+#
+# * Harms: overpolicing of neighborhoods can
+#   lead to disproportionate effect on communities in these neighborhoods
+#   (perhaps exacerbated by feedback loop)
+#
+# * Notes: Feedback loops!
+#   Prediction on behavior based on circumstances; perhaps useful for
+#   aggregate observations about behavior but less so for individual
+#   predictions
+#
+# * Dangerous to assume that ‘crime’ is a single phenomenon.
+#   There are different kinds of crime, so also need to make sure that data
+#   match the crime being predicted.
+#
+# For this case study, we are utilizing the Boston housing value dataset
+# (described here). Originally, this dataset includes the following variables:
+#
+# .. list-table::
+#    :header-rows: 1
+#    :widths: 5 20
+#    :stub-columns: 1
+#
+#    *  - Variable
+#       - Description
+#    *  - CRIM (BLUE)
+#       - per capita crime rate by town
+#    *  - ZN
+#       - proportion of residential land zoned for lots over 25,000 sq.ft.
+#    *  - INDUS
+#       - proportion of non-retail business acres per town
+#    *  - CHAS
+#       - Charles River dummy variable (= 1 if tract bounds river; 0 otherwise)
+#    *  - NOX
+#       - nitric oxides concentration (parts per 10 million)
+#    *  - RM
+#       - average number of rooms per dwelling
+#    *  - AGE
+#       - proportion of owner-occupied units built prior to 1940
+#    *  - DIS
+#       - weighted distances to five Boston employment centres
+#    *  - RAD
+#       - index of accessibility to radial highways
+#    *  - TAX
+#       - full-value property-tax rate per $10,000
+#    *  - PTRATIO
+#       - pupil-teacher ratio by town
+#    *  - B
+#       - 1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town
+#    *  - LSTAT (BLUE)
+#       - % lower status of the population
+#    *  - MEDV (GREEN)
+#       - Median value of owner-occupied homes in $1000's
+# *Figure A: Dataset variables*
+#
+# Where MEDV (colored green in Figure A above) is the target variable
+# (that a model built on this dataset is intended to predict). We make
+# modifications below with the understanding that LSTAT is a parameter of
+# concern.
+#
+# **Import and install relevant packages**
+#
+import numpy as np
+import pandas as pd
+from fairlearn.metrics import MetricFrame
+from fairlearn.metrics import selection_rate
+from fairlearn.reductions import DemographicParity, ExponentiatedGradient
+from fairlearn.datasets import fetch_boston
+from sklearn.metrics import accuracy_score
+from sklearn.tree import DecisionTreeClassifier
+from statistics import mean, median
+from fairlearn.widget import FairlearnDashboard
```

Comment:
The dashboard will be removed shortly. We'll need to replace whatever you're using it for with the new plots #561 or custom ones.

Reply:
Once the dashboard is removed, my understanding is I would do the following to add (as @kevinrobinson suggested) the new plots: 

**At the top** 
from fairlearn.plots import plot_disparities_in_performance, plot_disparities_in_selection_rate

**In the sections where I produced plots**
plot_disparities_in_performance(y_true, y_pred, _sensitive_feature_)

plot_disparities_in_selection_rate(y_true, y_pred, _sensitive_feature_)

Is my understanding correct? When should I anticipate his PR being merged?

Reply:
That's right! At this point we can merge it, but there are a few comments on the PR. I expect that we'll see some progress there before the end of the month, but need to check on it now.